beyond-essence/inventory/toggle-switches-updated.md

```diff
@@ -0,0 +1,127 @@
+
+Depending on the complexity of the feature or mode, additional content may be shown below the toggle switch.
+
+{% image_custom image="/assets/img/pages/essence/pricing-for-variations.png" width="100" %}
+
+For further information on the exact visualisation of addtional content, consult the design section.
+
+## Position
+
+Toggle switches may be positioned on a card together with further UI elements that relate to a specific topic.
+
+## Behavior
+
+By selecting either the switch itself or its label, it changes its state:
+
+{% image_custom image="/assets/img/pages/essence/switch-button.gif" width="100" %}
+
+## Choosing between toggle switches and checkboxes
+
+Toggle switches and checkboxes seem to be quite similar as they are both used to activate some sort of "settings".
+
+To decide when to use a toggle switch, here are some clues:
+
+* Use a toggle switch to allow merchants to enable features that have a significant effect on their workflow.
+* Use a toggle switch to enhance a card with additional content.
+* Don't use a toggle switch to activate small and distinct options, like an additional price field.
+* Don't use a toggle switch, when the merchant needs to select multiple items in a list that need to be saved afterwards.
+
+## Copy writing
+
+As toggle switches typically allow merchants to enable or disable features or modes that have a significant effect on their workflow and data might be lost when they deactivate a previously activated feature, make sure the label (and description, if applicable) are precise and make it easy for the merchant to understand the consequences of activating/deactivating the toggle.
+
+* Be short, precise and direct
+* Start the toggle label with a verb and describe what changing its default state will do, e.g. “Allow visitors to make purchases on your website.” (exception: “Website is live”)
+* If there is a static description text underneath the toggle, describe what will happen when the switch is activated.
+Start with “If this toggle is activated…”
+* If the description text only appears after the feature is toggled on, describe what will happen when the switch is turned off.
+Start with: “If you toggle off FEATURE…”
+* Don't use neutral or ambiguous phrases.
+* Avoid asking questions.
+
+**DO**: "If this toggle is activated, there will be a cart displayed in the header of your website as well as an "Add to cart" button on each product.
+Be sure to save any changes."
+
+_Precisely explains what happens when the switch is turned on._
+
+**DON'T**: "A cart will be displayed in the header of your website."
+
+_Too ambigous, unclear if this is what happens when the toggle is turned on or off_
+
+**ALSO DON'T**: "Want to display a cart in the header of your website?"
+
+_The question format makes in unclear for the merchant if they need to activate or deactivate the toggle for the desired outcome._
+
+## Design
```

Comment:
At the moment we are not really consistent but for sure I can say that we do not have the light beige background anymore 🙈

The new design includes a label and an explanation text. Depending on the functionality only a label is visible, after toggling the switch of the explanation is also visible.

The Default is a label and an explanation. The toggle switch with more complexity like for variation handling has more design elements. In addition, such a complex toggle switch using a confirmation dialog.

Design 1
![image](https://user-images.githubusercontent.com/13624807/98004113-d30a7800-1def-11eb-8446-8e88a2a63c9c.png)

Design 2
![image](https://user-images.githubusercontent.com/13624807/98004152-dd2c7680-1def-11eb-9329-9a6015848b52.png)

Design 3
![image](https://user-images.githubusercontent.com/13624807/98004194-eae1fc00-1def-11eb-8039-081da78fed53.png)

I have created a ticket to align the content of the toggle switch which would help to define a more precise look and feel.

Reply:
@ywolpers shall we replace the light background as a quick fix with a white background? 😬

Reply:
@katjamueller When reviewing, I noticed that the "Design" and the "Structure" section have almost the same content, so I consolidated them into one section that also should be matching your explanation here in the comment. Would you mind taking another look if that looks ok as MVP version?